plugin-api/src/main/java/org/sonarsource/sonarlint/plugin/api/issue/NewQuickFix.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * SonarLint Plugin API
+ * Copyright (C) 2016-2021 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.sonarlint.plugin.api.issue;
+
+/**
+ * Describe a quick fix for a {@link NewSonarLintIssue}, with a description and a collection of {@link NewInputFileEdit}.
+ * Input file edits will be applied in the order they are added, insofar that they are compatible with one another.
```

Comment:
Do we need to restate this in this class ? Or is it about order of QuickFixes ?

Reply:
Yes, I believe it deserves to be explicit: input file edits of a given quick fix are applied in the order they are declared, and text edits of a given input file edit are applied in the order they are declared.